File: algorithms/1832.check-if-the-sentence-is-pangram/solution.go
func checkIfPangram(sentence string) bool {
	m := map[rune]int{}
	for _, r := range sentence {
		if _, ok := m[r]; ok {
			m[r] += 1
		} else {
			m[r] = 1
		}
	}
	return len(m) == 26
}

File: algorithms/1832.check-if-the-sentence-is-pangram/solution.rs
impl Solution {
    pub fn check_if_pangram(sentence: String) -> bool {
        // sort & dedup
        // let mut letters: Vec<char> = sentence.chars().collect();
        // letters.sort_unstable();
        // letters.dedup();
        // letters.len() == 26

        // using Hash
        use std::collections::HashMap;
        let mut map: HashMap<char, i32> = HashMap::new();
        for c in sentence.chars() {
            let count = map.entry(c).or_insert(0);
            *count += 1
        }
        map.keys().len() == 26
    }
}
